tests/integration/tls/test_tls.py

```diff
@@ -86,3 +94,63 @@ async def test_cluster_formation_after_tls(ops_test: OpsTest) -> None:
     leader_unit_ip = await get_leader_unit_ip(ops_test)
 
     assert await check_cluster_formation_successful(ops_test, leader_unit_ip, unit_names)
+
+
+@pytest.mark.runner(["self-hosted", "linux", "X64", "jammy", "large"])
+@pytest.mark.group(1)
+@pytest.mark.abort_on_fail
+async def test_tls_renewal_without_restart(ops_test: OpsTest) -> None:
```

Comment:
Since we can't really check if there was a restart or not, I'd call this test `test_tls_renewal` or so.
As in reality, _that's_ what it is testsing

Reply:
Done